tests/manual/iot-core/publish-from-registry-to-device-commands.sh

```diff
@@ -1,14 +1,19 @@
-#fixme add asserts that env variables are defined
+#!/bin/sh
 
-#REGISTRY_ID="areiqldr2chj5v0g2rd6"
-#REGISTRY_PASSWORD="zx+z{i1QD6YJ#J#&"
-#DEVICE_ID="arekifdlttq12q6sslle"
-#DEVICE_PASSWORD="HsTvGAN6HFundl1v"
+REGISTRY_ID=$(echo "$IOT_REGISTRY")
+REGISTRY_PASSWORD=$(echo "$IOT_REGISTRY_PASSWORD")
 
+EMPTY_LINE=""
+
+echo "--------------------------------------------------------"
+echo "Credentials: '${REGISTRY_ID}' : '${REGISTRY_PASSWORD}'"
```

Comment:
@dmitry-osin 
does it make sense to hide the password? 

might we have the following case?
if we forget to remove it and this script is used in the github actions then we might compromise the test cloud folder registry.

as result, a bad person can just spam mqtt messages and increase cloud costs

Reply:
Thanks for reply. I believe that these values will be taken from dev machine envs and you do not need to modify this script.